homeassistant/components/lcn/services.py

```diff
@@ -361,7 +354,7 @@ async def async_call_service(self, service):
         else:
             await device_connection.lock_keys(table_id, states)
 
-        handler = device_connection.status_request_handler
+        handler = device_connection.status_requests_handler
```

Comment:
Is this a bug fix?

Reply:
Actually yes. I changed the PR comment.
Or do you want me to make a separate PR for this?

Reply:
If we want to release it in a patch release it should go in a separate PR. Otherwise it's ok, although generally we want to separate clean ups from bug fixes.

Reply:
Ok, I will keep that in mind in the future and leave it here as it is.